net/net_evbuffer.c

```diff
@@ -3978,6 +3989,14 @@ int get_hosts_metric(const char *netname, enum net_metric_type type)
     return metric.value;
 }
 
+int revcon_should_reject(int admin)
+{
+    intptr_t ret;
```

Comment:
Don't need the pointer anymore. Just use the `int` field in `should_reject_request_arg` struct directly.

Reply:
yes - tx for review ..